lib/core/buildFullPath.js

```diff
@@ -10,10 +10,13 @@ var combineURLs = require('../helpers/combineURLs');
  *
  * @param {string} baseURL The base URL
  * @param {string} requestedURL Absolute or relative URL to combine
+ * @param {boolean} [allowAbsoluteUrls=false] Set to true to allow absolute URLs
```

Comment:
this implies that the default is `false`, but it's actually `true` (since we're checking for explicit `false` in the function)